docs/_posts/murat-gunay/2021-12-31-nerdl_restaurant_100d_en.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: model
+title: Detect Restaurant-related Terminology
+author: John Snow Labs
+name: nerdl_restaurant_100d
+date: 2021-12-31
+tags: [ner, restaurant, en, open_source]
+task: Named Entity Recognition
+language: en
+edition: Spark NLP 3.3.4
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is trained with `glove_100d` embeddings to detect restaurant-related terminology.
+
+## Predicted Entities
+
+`Location`, `Cuisine`, `Amenity`, `Restaurant_Name`, `Dish`, `Rating`, `Hours`, `Price`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/nerdl_restaurant_100d_en_3.3.4_3.0_1640949258750.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
+
+embeddings = WordEmbeddingsModel.pretrained("glove_100d") \
+      .setInputCols("sentence", "token") \
+      .setOutputCol("embeddings")
+
+nerdl = NerDLModel.pretrained("nerdl_restaurant_100d")\
+    .setInputCols(["sentence", "token", "embeddings"])\
+    .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, embeddings, nerdl, ner_converter])
+
+text = """2 star restaurants with inside dining and 5 star resturants are in my town. Besides that, Hong Kong restaurant reasonable prices a great lunch spot but open till 2 am, called Passims Kitchen."""
```

Comment:
The example is not native English, has several grammatical mistakes. Was it taken from the training or test set? Here you can find a good article about a restaurant review in native English: https://www.sassyhongkong.com/eat-drink-lunch-deals-set-menus-restaurants/

Reply:
@josejuanmartinez 
Yes you may be right. The text was grammatically not so good but it was a native customer comments from train data set. Sometimes, you know, humans don't obey grammar... For the best model result we generally take text from train set as a best practice.